src/pharmpy/model/statements.py

```diff
@@ -1416,6 +1557,41 @@ def amount(self):
 
     @property
     def dose(self):
+        if self._dose is not None:
+            return tuple(sorted(self._dose, key=lambda d: d.admid))
+        else:
+            return self._dose
+
+    @property
+    def oral_dose(self):
+        if self._dose is not None:
+            oral_dose = tuple([dose for dose in self.dose if dose.admid == 1])
+            return oral_dose if oral_dose else None
+        return None
+
+    @property
+    def iv_dose(self):
+        if self._dose is not None:
+            iv_dose = tuple([dose for dose in self.dose if dose.admid == 2])
+            return iv_dose if iv_dose else None
+        return None
```

Comment:
Are these used anywhere? Maybe it's better to wait to add these methods until they're needed?

Reply:
Do we even have a way of distinguishing the oral dose from the iv?

Reply:
This was done base on admid and where used in some special places for ODE transformations. However, as they are not necessarily needed, they have been removed for now.